### PR TITLE
mc-release-4.10.0

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
mc-release-4.10.0

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by johnbenjamin-hmcts — DISMISSED** [review]
Approved

**Review by Mo-Lala84 — DISMISSED** [review]
approved

**Review by ernestman28 — DISMISSED** [review]

**Review by Mo-Lala84 — DISMISSED** [review]
approved

**Review by Surekha-HMCTS — DISMISSED** [review]
Approved

**Review by Mo-Lala84 — DISMISSED** [review]
approved

**Review by udaydenduluri33 — APPROVED** [review]
@Mo-Lala84 Thanks for the changes

**Review by Kartheek-S — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by phillip-whitaker-hmcts — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "@angular/router": "^7.2.13",
     "@hmcts/ccpay-web-component": "5.0.4",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "4.13.8",
+    "@hmcts/ccd-case-ui-toolkit": "4.10.0-rc.6",
```
Comment (rayliangatos) [inline package.json:75]: could this be a live version instead of rc?

Comment (Mo-Lala84) [reply package.json:75]: its live yes but i will make a live tag and import it as business is driving the changes needed

**Review by tarikdogan-hmcts — APPROVED** [review]

**Pull request merged**